submissions/expero1/js-dom/script.js

```diff
@@ -0,0 +1,82 @@
+const createElem = (elem, attrObj, className) => {
```

Comment:
I guess you should choose one strategy for setting class. Because in some cases you set it by `className` in other by `setAttribute`, and in others you use both `className` argument and `class` inside your `attrObj`

(Personally I prefer `classList`)

So,  choose one strategy, please.

Reply:
Thanks. Fixed.